app/controllers/application_controller.rb

```diff
@@ -51,6 +57,12 @@ def set_seen_cookie_message
     cookies[:seen_cookie_message] = { value: 'yes', expires: 1.year.from_now, httponly: true }
   end
 
+  def set_cors_headers
+    headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
Are we sure we want to allow any domain here? There isn't going to be any performance impact by opening the API to be loaded on any domain?